caput/scripts/runner.py

```diff
@@ -190,8 +194,11 @@ def template_run(templatefile, var):
             tfh.write(expanded_string)
             tfh.flush()
 
-            # Run the pipeline
-            Manager.from_yaml_file(tfh.name).run()
+            # Run or queue the pipeline
+            if queue:
+                ctx.invoke(queue, configfile=tfh.name)
```

Comment:
Do you need to pass `submit=True`?

Reply:
Yeah, the issue is that it should be `if submit` !